<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyerTorrentPluginConfigurationDialog</class>
 <widget class="QDialog" name="BuyerTorrentPluginConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buyer</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>193</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="numPeersLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>#Seeders:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numPeersLineEdit">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>230</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxLockLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum lock time (h:m):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxTotalSpendLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>24</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="maxLockTimeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="maxTotalSpendLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>24</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum total spend (*):</string>
   </property>
  </widget>
  <widget class="QLabel" name="waitConfirmationTimeLabel">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>328</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum wait time (h:m):</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="maxConfirmationTimeTimeEdit">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>326</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>1</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="feeLabel">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fee/kB  (*):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="feePrKbLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Required value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalValueLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuyerTorrentPluginConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuyerTorrentPluginConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
